chains/evm/listener/event-handler_test.go

```diff
@@ -0,0 +1,61 @@
+package listener
+
+import (
+	"math/big"
+	"reflect"
+	"testing"
+
+	"github.com/ChainSafe/chainbridge-core/relayer"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/math"
+)
+
+func TestErc20EventHandler(t *testing.T) {
```

Comment:
We should probably be using test suites in tests to be consistent like https://github.com/ChainSafe/chainbridge-celo/blob/main/chain/listener/listener_test.go#L32

I think it should be probably called Erc20HandlerTestSuite to make it more easier to find tests associated with that code

Reply:
Should probably also add test for when calldata len is zero to have 100% coverage :smile: